.github/workflows/cicd-main.yml

```
@@ -2080,6 +2080,16 @@ jobs:
         rm -rf /tmp/output_nemo2_ckpt
         rm -rf /tmp/output_hf_ckpt
 
+  # L2: Tron
+  L2_MLM_Llama1B_Train:
```

let's name it tron for now. Using MLM as a name will be confusing since we don't take a direct dependence on it.

```suggestion
  L2_Tron_Llama1B_Train:
```

---

+1